pkgdb/tests/data/realise/lockfiles/single-package/manifest.lock

```diff
@@ -0,0 +1 @@
-darwin","x86_64-darwin","aarch64-linux","x86_64-linux"]},"registry":{"defaults":{"subtrees":null},"inputs":{"nixpkgs":{"from":{"owner":"NixOS","repo":"nixpkgs","rev":"ab5fd150146dcfe41fda501134e6503932cc8dfd","type":"github"},"subtrees":["legacyPackages"]}},"priority":["nixpkgs"]}},"packages":{"aarch64-darwin":{"vim":{"attr-path":["legacyPackages","aarch64-darwin","vim"],"info":{"broken":false,"description":"The most popular clone of the VI editor","license":"Vim","pname":"vim","unfree":false,"version":"9.0.2048"},"input":{"attrs":{"lastModified":1704300003,"narHash":"sha256-FRC/OlLVvKkrdm+RtrODQPufD0vVZYA0hpH9RPaHmp4=","owner":"NixOS","repo":"nixpkgs","rev":"ab5fd150146dcfe41fda501134e6503932cc8dfd","type":"github"},"fingerprint":"9bb3d4c033fbad8efb5e28ffcd1d70383e0c5bbcb7cc5c526b824524467b19b9","url":"github:NixOS/nixpkgs/ab5fd150146dcfe41fda501134e6503932cc8dfd"},"priority":5}},"aarch64-linux":{"vim":{"attr-path":["legacyPackages","aarch64-linux","vim"],"info":{"broken":false,"description":"The most popular clone of the VI editor","license":"Vim","pname":"vim","unfree":false,"version":"9.0.2048"},"input":{"attrs":{"lastModified":1704300003,"narHash":"sha256-FRC/OlLVvKkrdm+RtrODQPufD0vVZYA0hpH9RPaHmp4=","owner":"NixOS","repo":"nixpkgs","rev":"ab5fd150146dcfe41fda501134e6503932cc8dfd","type":"github"},"fingerprint":"9bb3d4c033fbad8efb5e28ffcd1d70383e0c5bbcb7cc5c526b824524467b19b9","url":"github:NixOS/nixpkgs/ab5fd150146dcfe41fda501134e6503932cc8dfd"},"priority":5}},"x86_64-darwin":{"vim":{"attr-path":["legacyPackages","x86_64-darwin","vim"],"info":{"broken":false,"description":"The most popular clone of the VI editor","license":"Vim","pname":"vim","unfree":false,"version":"9.0.2048"},"input":{"attrs":{"lastModified":1704300003,"narHash":"sha256-FRC/OlLVvKkrdm+RtrODQPufD0vVZYA0hpH9RPaHmp4=","owner":"NixOS","repo":"nixpkgs","rev":"ab5fd150146dcfe41fda501134e6503932cc8dfd","type":"github"},"fingerprint":"9bb3d4c033fbad8efb5e28ffcd1d70383e0c5bbcb7cc5c526b824524467b19b9","url":"github:NixOS/nixpkgs/ab5fd150146dcfe41fda501134e6503932cc8dfd"},"priority":5}},"x86_64-linux":{"vim":{"attr-path":["legacyPackages","x86_64-linux","vim"],"info":{"broken":false,"description":"The most popular clone of the VI editor","license":"Vim","pname":"vim","unfree":false,"version":"9.0.2048"},"input":{"attrs":{"lastModified":1704300003,"narHash":"sha256-FRC/OlLVvKkrdm+RtrODQPufD0vVZYA0hpH9RPaHmp4=","owner":"NixOS","repo":"nixpkgs","rev":"ab5fd150146dcfe41fda501134e6503932cc8dfd","type":"github"},"fingerprint":"9bb3d4c033fbad8efb5e28ffcd1d70383e0c5bbcb7cc5c526b824524467b19b9","url":"github:NixOS/nixpkgs/ab5fd150146dcfe41fda501134e6503932cc8dfd"},"priority":5}}},"registry":{"defaults":{"subtrees":null},"inputs":{"nixpkgs":{"from":{"lastModified":1704300003,"narHash":"sha256-FRC/OlLVvKkrdm+RtrODQPufD0vVZYA0hpH9RPaHmp4=","owner":"NixOS","repo":"nixpkgs","rev":"ab5fd150146dcfe41fda501134e6503932cc8dfd","type":"github"},"subtrees":["legacyPackages"]}},"priority":["nixpkgs"]}}
```

Comment:
I think we are actively removing support for v0 manifests and lockfiles so its a bit surprising to see `"lockfile-version": 0` being used here.

(non-blocking)
My suggestion would be to upgrade this to v1, now or in a following iteration.

Reply:
Agreed; I literally copied all test data from the buildenv files and modified as required, and this overlapped Zach's efforts to retire all v0 test data. We will need to similarly re-sweep that test data once this gets merged.